### DISCUSSION TITLE
camera updating every second

### DISCUSSION THREAD

**Discussion by starschild86** [greghesp/ha-bambulab#449] [General]
hi all,
first of all congratulations on the great addon, it really is cool!
I especially like the ability to create an automation to shut down the light after printing (i was not able to do so using gcode on my p1p) and the ability to see the webcam when i'm not home (i didn't want to open ports on my printer).
i have a request however. the log is currently flooded by updates on cam, would it be possible to disable them?
also is the cam uploading a picture every second? does this consume bandwith/ create stress on my home assistant device? would it be possible to have it active only when a print is in progress?
best,
Nicola

**Comment by gregsmith-movista** [comment]
I came here to second this... I too have the P1P and the constant update from the camera every seconds in the logbook is maddening. I makes finding anything else in there so much more difficult than is needed.

**Comment by AdrianGarside** [comment]
See https://github.com/greghesp/ha-bambulab/issues/423
